### ISSUE TITLE
Error when assembling firmware

### ISSUE THREAD

**Issue by Artem0000001** [bigtreetech/BIGTREETECH-TouchScreenFirmware#2926]
Error when assembling the firmware, I'm running Visual Studio Code 1.88.0 PlatformIO Core 6.1.14·Home 3.4.4
I can't figure out what the problem is

c:/.platformio/packages/toolchain-gccarmnoneeabi/bin/../lib/gcc/arm-none-eabi/9.2.1/../../../../arm-none-eabi/bin/ld.exe: cannot open linker script file C:/Users/КеК/OneDrive/Рабочий стол/BIGTREETECH-TouchScreenFirmware-master/buildroot/ldscripts/gd32f20xxC_0x3000_iap.ld: Invalid argument
collect2.exe: error: ld returned 1 exit status
*** [.pio\build\BIGTREE_GD_TFT35_E3_V3_0\BIGTREE_GD_TFT35_V3.0_E3.27.x.elf] Error 1 
![oN2DVFbL7RY](https://github.com/bigtreetech/BIGTREETECH-TouchScreenFirmware/assets/166506915/2bddfffb-40dd-46f6-8714-798488d85717)
![sj8OC3vYzfE](https://github.com/bigtreetech/BIGTREETECH-TouchScreenFirmware/assets/166506915/93ef5c48-350c-49ea-8faf-cb7c50d330a9)

**Comment by rondlh** [comment]
This issue is caused by corruption, did you recently go into places where you don't belong, I recommend to get out of there asap!

**Comment by Artem0000001** [comment]
> This issue is caused by corruption, did you recently go into places where you don't belong, I recommend to get out of there asap!

I do not understand you. I just want to customize the btt tft35 v3.0.1 screen. I have little knowledge of programming skills

**Comment by thisiskeithb** [comment]
This is likely due to special characters in your file path (ex: `Рабочий стол`). Move the firmware to a folder close to the root of your hard drive (like `C:\firmware`) and try to compile again.

**Comment by kisslorand** [comment]
> This issue is caused by corruption, did you recently go into places where you don't belong, I recommend to get out of there asap!

I agree. They should move everything from `C:\Украина` to `C:\Россия` and things will be fine.
